datis.php

```diff
@@ -204,8 +204,15 @@
         $out_dewpt_data = $str_dewpt_data ;
 }
 
-print(' T ' . $out_temp_data . ' /DP ' . $out_dewpt_data . ' QNH ' . $decoded->getPressure()->getValue() . ' HPA ');
+print(' TEMP ' . $out_temp_data . ' DEW POINT ' . $out_dewpt_data . ' QNH ' . $decoded->getPressure()->getValue() . ' HPA ');
 
-print ('RPT RECEIPT OF ATIS ' . $_GET['info'] . ' ON ' . $decoded->getIcao());
+if (empty($phenomenon)) {
+} else {
+    if (in_array("DZ", $phenomenon[0]->getTypes()) or in_array("RA", $phenomenon[0]->getTypes()) or in_array("SN", $phenomenon[0]->getTypes()) or in_array("SG", $phenomenon[0]->getTypes())) {
```

Comment:
Use `||` instead of `or` where possible. And split the condition into multiple lines if the line gets too long. For example,
```php
if ((condition1)
    || (condition2)
    || (condition3)
) {
    // do something
}
```